### PR TITLE
add one

### PR DESCRIPTION
**Pull request by lizhichao**

### CODE REVIEW

File: frameworks/PHP/one/App/Controllers/IndexController.php [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+    public function fortunes()
+    {
+        $arr   = Fortune::findAll()->toArray();
+        $rs    = [];
+        $rs[0] = 'Additional fortune added at request time.';
+        foreach ($arr as $item) {
+            $rs[$item['id']] = $item['message'];
+        }
+        asort($rs);
```
Comment (joanhey) [inline frameworks/PHP/one/App/Controllers/IndexController.php:42]: Please, use objects to maintain the full ORM.
Now it's using arrays.
https://github.com/TechEmpower/FrameworkBenchmarks/wiki/Project-Information-Framework-Tests-Overview#fortunes

Comment (lizhichao) [reply frameworks/PHP/one/App/Controllers/IndexController.php]: `Fortune::findAll()`  is objects to maintain the full ORM

Comment (joanhey) [reply frameworks/PHP/one/App/Controllers/IndexController.php]: For me, I don't see any object.
All is managed like array and the `asort` only order arrays.

Comment (lizhichao) [reply frameworks/PHP/one/App/Controllers/IndexController.php]: `0e78a78` commits , The parameter of `asort` is not `Fortune::findAll()`

Comment (matt-42) [reply frameworks/PHP/one/App/Controllers/IndexController.php]: Using usort directly on the $arr array of objects and removing the $rs key value map would fit the requirement. Here is an example:
https://github.com/TechEmpower/FrameworkBenchmarks/blob/master/frameworks/PHP/kumbiaphp/bench/app/controllers/fortunes_controller.php#L10

**Comment by lizhichao** [comment]
done

**Pull request merged**